core/src/main/java/org/apache/accumulo/core/file/rfile/RFile.java

```diff
@@ -993,7 +992,7 @@ private void _seek(Range range) throws IOException {
           // causing the build of an index... doing this could slow down some use cases and
           // and speed up others.
 
-          MutableByteSequence valbs = new MutableByteSequence(new byte[64], 0, 0);
+          ArrayByteSequence valbs = new ArrayByteSequence(new byte[64], 0, 0);
```

Comment:
For the lines that changed, you could use `var` to make it slightly cleaner, since the type is redundant in these declarations.

```suggestion
          var valbs = new ArrayByteSequence(new byte[64], 0, 0);
```

Reply:
Done, updated.